projects/fsastorefrontlib/src/cms-components/checkout/components/add-options/add-options.component.ts

```diff
@@ -23,12 +28,14 @@ export class AddOptionsComponent implements OnInit, OnDestroy {
     protected routingService: RoutingService,
     protected checkoutConfigService: FSCheckoutConfigService,
     protected activatedRoute: ActivatedRoute,
-    protected currencyService: CurrencyService
+    protected currencyService: CurrencyService,
+    protected userIdService: UserIdService
   ) {}
 
   entries$: Observable<OrderEntry[]>;
   isCartStable$: Observable<boolean>;
   subscription = new Subscription();
+  chatbotSubscription = new Subscription();
```

Comment:
Is there need to split subscriptions? If not then just simply add chatbot to already existing `subscription`